main.go

```diff
@@ -6,16 +6,22 @@ package main
 
 import (
 	"context"
-
-	"github.com/owner/name/plugin"
-
+	"github.com/drone/drone-nuget/plugin"
+	"github.com/joho/godotenv"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/sirupsen/logrus"
 )
 
 func main() {
 	logrus.SetFormatter(new(formatter))
 
+	// load env variables
+	err := godotenv.Load(".env")
+
```

Comment:
you can remove the space.  also it would be valid to shorten (although this is optional)

```
if err := godotenv.Load(".env"); err != nil {
  ...
}

```

Reply:
I decided to remove this block of code rather than trying to go down the route of conditional conditional compilation. I could have just added a debug export to my shell and used that if it existed. But also decided against it.